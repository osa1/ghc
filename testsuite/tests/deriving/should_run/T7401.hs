{-# LANGUAGE EmptyDataDecls #-}

data Foo deriving (Show, Read, Eq, Ord)

main = print $ (undefined :: Foo) == (undefined :: Foo)
