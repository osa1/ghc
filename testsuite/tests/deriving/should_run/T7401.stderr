T7401: == on empty data type Foo
